doc/integrator/create_application.rst

```diff
@@ -65,8 +65,14 @@ Then you should checkout the branch or tag of the version you want to install:
 
     git checkout <branch|tag>
 
-``<branch|tag>`` can be ``1.4`` for the latest version of the 1.4 branch,
-``1.4.0`` for the first stable 1.4 version.
+``<branch|tag>`` can be ``2.1`` for the latest version of the 2.1 branch,
```

Comment:
./docker-run script is only available on master branch maybe we can replace 2.1 by master or future 2.2 ?